sdk/js/src/utils/repairVaa.ts

```diff
@@ -17,14 +15,19 @@ interface GuardianSetData {
 }
 
 export async function getCurrentGuardianSet(
-  provider: ethers.providers.JsonRpcProvider
+  provider: ethers.providers.JsonRpcProvider,
+  chainId: ChainId,
```

Comment:
I think we should add support for non-EVM chains or at least set the type here to `EVMChainId`.